reference/configuration/framework.rst

```diff
@@ -2938,9 +2938,9 @@ The cache clearer used to clear your PSR-6 cache.
 prefix_seed
 ...........
 
-**type**: ``string`` **default**: ``null``
+**type**: ``string`` **default**: ``_%kernel.project_dir%.%kernel.container_class%``
```

Comment:
see https://github.com/symfony/symfony/blob/5.2/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L983 (it has already been the default before 5.2, just implicitly in code, not in the configuration)